apps/ecs/behaviours/send-email-notification.js

```diff
@@ -0,0 +1,141 @@
+      req.sessionModel.get('worker-been-in-uk-before-1988-full-name'),
+    worker_date_birth: moment(req.sessionModel.get('worker-dob')).format(PRETTY_DATE_FORMAT) ??
+      moment(req.sessionModel.get('worker-been-in-uk-before-1988-dob')).format(PRETTY_DATE_FORMAT),
+    worker_nationality: req.sessionModel.get('worker-nationality') ??
+      req.sessionModel.get('worker-been-in-uk-before-1988-nationality'),
+    worker_address: req.sessionModel.get('workerAddress') ?? req.sessionModel.get('workerUkAddress'),
+    has_worker_country: req.sessionModel.get('steps').includes('/worker-address') ? 'yes' : 'no',
+    worker_country: req.sessionModel.get('worker-country') ?? '',
+    job_title: req.sessionModel.get('job-title'),
+    hours_of_work_per_week: req.sessionModel.get('hours-of-work-per-week'),
+    person_work_for_you: req.sessionModel.get('person-work-for-you'),
+    person_worked_for_you: req.sessionModel.get('person-work-for-you') === 'yes' ? 'yes' : 'no',
+    worker_start_date: req.sessionModel.get('start-work-date') ?? '',
+    work_as_result_of_tupe_transfer: req.sessionModel.get('work-for-you-result-of-tupe-transfer') ?? '',
+    result_of_tupe_transfer: req.sessionModel.get('work-for-you-result-of-tupe-transfer') === 'yes' ? 'yes' : 'no',
+    tupe_transfer_date: req.sessionModel.get('tupe-date') ?? '',
+    use_digital_right_to_work: req.sessionModel.get('use-digital-right-to-work'),
+    digital_right_to_work: req.sessionModel.get('use-digital-right-to-work') === 'no' ? 'yes' : 'no',
+    applied_for_eu_settlement: req.sessionModel.get('worker-applied-eu-settlement-scheme') ?? '',
+    has_eu_settlement: req.sessionModel.get('use-digital-right-to-work') === 'no' &&
+      req.sessionModel.get('worker-applied-eu-settlement-scheme') === 'none-of-above' ? 'yes' : 'no',
+    worker_has_arc_card: req.sessionModel.get('worker-has-arc-card') ?? '',
+    has_arc_card: req.sessionModel.get('use-digital-right-to-work') === 'no' &&
+      req.sessionModel.get('worker-has-arc-card') === 'yes' ? 'yes' : 'no',
+    seen_original_document: req.sessionModel.get('seen-original-document') ?? '',
+    arc_number: req.sessionModel.get('arc-number') ?? '',
+    not_have_arc_card: req.sessionModel.get('use-digital-right-to-work') === 'no' &&
+      req.sessionModel.get('worker-has-arc-card') === 'no' ? 'yes' : 'no',
+    worker_have_ongoing_appeal: req.sessionModel.get('worker-have-ongoing-appeal') ?? '',
+    not_have_ongoing_appeal: req.sessionModel.get('use-digital-right-to-work') === 'no' &&
+      req.sessionModel.get('worker-have-ongoing-appeal') === 'no' ? 'yes' : 'no',
+    worker_been_in_UK_before_1988: req.sessionModel.get('worker-been-in-UK-before-1988') ?? '',
+    has_worker_details_1988: req.sessionModel.get('steps').includes('/worker-details-1988') ? 'yes' : 'no',
+    place_of_birth: req.sessionModel.get('worker-place-of-birth') ?? '',
+    Year_of_entry_to_uk: req.sessionModel.get('worker-year-of-entry-to-uk') ?? '',
```

Comment:
lowercase `Y` for name convention
this will also need to be updated on the GOV Notify template